Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -49,6 +48,7 @@ def validate_email_sent(incident_id, email_subject, subject_include_incident_id,
         subject_include_incident_id: Should we include the incident id in the email subject.
         email_to: The email's recipients.
         reply_body: The email body.
+        body_type: The body type of the email.
```

Comment:
In several places.
Also - Since it's a closed group of values (`Text`, `HTML`,) then maybe consider using an enum?
```suggestion
        body_type: The type of the email body.
```